qa-dcp.planx-pla.net/portal/gitops.json

```diff
@@ -335,13 +335,13 @@
         { "field": "rbc_ncnc_bld", "name": "Red Blood Cells Number Concentration Bld" },
         { "field": "rdw_ratio_rbc", "name": "Red Cell Distribution Width Ratio Rbc" },
         { "field": "wbc_ncnc_bld", "name": "White Blood Cells Number Concentration Bld" },
-        { "field": "bmi_baseline", "name": "BMI Baseline"},
-        { "field": "current_smoker_baseline", "name": "Current Smoker Baseline"},
-        { "field": "ever_smoker_baseline", "name": "Ever Smoker Baseline"},
-        { "field": "geographic_site", "name": "Geographic Site"},
-        { "field": "height_baseline", "name": "Height Baseline"},
-        { "field": "total_cholesterol", "name": "Total Cholesterol"},
-        { "field": "triglycerides", "name": "Triglycerides"}
+        { "field": "bmi_baseline", "name": "BMI Baseline" },
+        { "field": "current_smoker_baseline", "name": "Current Smoker Baseline" },
+        { "field": "ever_smoker_baseline", "name": "Ever Smoker Baseline" },
+        { "field": "geographic_site", "name": "Geographic Site" },
+        { "field": "height_baseline", "name": "Height Baseline" },
+        { "field": "total_cholesterol", "name": "Total Cholesterol" },
+        { "field": "triglycerides", "name": "Triglycerides" }
```

Comment:
Guppy will auto upper case the first character in each word separated by `_`, so a lot of these mappings are optional

the actual problem in `qa-dcp` is no one has run ETL after the etlmapping updated, hence none of these new fields are in the es index

i can help to run ETL in there